relay-monitors/src/lib.rs

```diff
@@ -135,35 +133,17 @@ pub fn process_check_in(payload: &[u8]) -> Result<Vec<u8>, ProcessCheckInError>
         check_in.status = CheckInStatus::Unknown;
     }
 
-    check_in.monitor_slug = slugify(&check_in.monitor_slug);
+    if let Some((overflow, _)) = check_in.monitor_slug.char_indices().nth(SLUG_LENGTH) {
+        check_in.monitor_slug.truncate(overflow);
+    }
```

Comment:
Just so I understand, this means that the slug is expected to contain non-ASCII text and the max length is given in unicode code points?

Reply:
@loewenheim yes. Do you know of a simpler way to truncate a string to a number of characters instead of bytes?

Reply:
No, I think it's fine this way. Just wanted to make sure that was the intended logic :)